src/cli/init.rs

```diff
@@ -8,13 +8,16 @@ pub struct Init {
     /// The app configuration file.
     #[clap(parse(from_os_str), long, default_value = DEFAULT_CONFIG_FILE)]
     pub config: std::path::PathBuf,
+
+    #[clap(parse(from_os_str), parse(try_from_str), default_value = "false")]
```

Comment:
Init does not need the expose option. Also, it should not call the `generate_files` function but the `generate_docker_files` (not linked with this issue...)